pkg/enforce/enforce.go

```diff
@@ -87,11 +87,10 @@ func EnforceAll(ctx context.Context, ghc *ghclients.GHClients) error {
 			opt.Page = resp.NextPage
 		}
 		for _, r := range repos {
-			if config.IsBotEnabled(ctx, ic, *r.Owner.Login, *r.Name) {
-				err := RunPolicies(ctx, ic, *r.Owner.Login, *r.Name)
-				if err != nil {
-					return err
-				}
+			enabled := config.IsBotEnabled(ctx, ic, *r.Owner.Login, *r.Name)
```

Comment:
Why do you need to check enabled bit seperately, why not do this inside runpolicies that gets all these args.

Reply:
It was just for ease of unit testing.